core/src/main/scala/kafka/tools/DumpLogSegments.scala

```diff
@@ -189,7 +189,7 @@ object DumpLogSegments {
       val wrapperMessageOpt = shallowIter.find(_.offset >= entry.offset + timeIndex.baseOffset)
       if (!wrapperMessageOpt.isDefined || wrapperMessageOpt.get.offset != entry.offset + timeIndex.baseOffset) {
```

Comment:
@hachikuji : Since we are focussing more on readability I have updated` if/else` to `match`  Please have a look and let me know if any further change is required.